### PR TITLE
Fix sidebar background brush, make HC work, and remove old style

### LINKED ISSUE #5025: Acrylic coloring issue in light mode.
**Explain the bug**
When interacting with certain UI elements in the app, the light mode acrylic occasionally switches themes to the dark mode tint.

**To reproduce**
Truthfully, I have no idea how this is reproduced exactly, however it often occurs when interacting with certain clickable UI elements. Two of these examples that trigger the visual bug almost 100% of the time in my testing were the TabView area after opening a new tab, and selecting a file entry in Details View.

**Expected behavior**
The acrylic keeps it's light coloring.

**Logfile**
N/A

**Screenshots**

https://user-images.githubusercontent.com/42101043/119436496-df014d00-bce1-11eb-87b0-f3e8f0220a29.mp4

**System Information**
 - OS Version: Dev (OS Build 21364.1000)
 - App version: 1.3.1.0 - **NOTE:** This is a built version of files from the `main` branch. I last pulled around 30 minutes before opening this issue. I'm not sure how version strings are handled on the developer build of Files.

**Additional comment**
None.

### PR DESCRIPTION
**Pull request by duke7553**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5025 

**Details of Changes**
Fix sidebar background brush issue, make HC work, and remove old style

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

**Review by winston-de — DISMISSED** [review]

File: Files/UserControls/NavigationToolbar.xaml [hunk 1]
```diff
@@ -282,105 +272,55 @@
...
+										</VisualStateManager.VisualStateGroups>
+									</Grid>
+								</ControlTemplate>
+							</Setter.Value>
+						</Setter>
+					</Style>
+				</ResourceDictionary>
+			</ResourceDictionary.MergedDictionaries>
+		</ResourceDictionary>
+	</UserControl.Resources>
```
Comment (yaichenbaum) [inline Files/UserControls/NavigationToolbar.xaml:324]: Can you fix the formatting in this file?

File: Files/App.xaml [hunk 2]
```diff
@@ -104,11 +115,11 @@
                             <x:Double x:Key="SidebarTintOpacity">0.8</x:Double>
                             <x:Double x:Key="SidebarTintLuminosityOpacity">0.8</x:Double>
                             <!--  Tabs  -->
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackground" Color="{StaticResource SystemControlBackgroundBaseLow}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPressed" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPointerOver" Color="{StaticResource SystemControlHighlightChromeHigh}" />
-                            <SolidColorBrush x:Key="TabViewItemSeparator" Color="{StaticResource SystemControlForegroundBaseLow}" />
```
Comment (yaichenbaum) [inline Files/App.xaml:111]: Is there a visual change?

File: Files/UserControls/NavigationToolbar.xaml (hunk 1 above)
Comment (lukeblevins) [reply Files/UserControls/NavigationToolbar.xaml]: @yair100 I can't fix formatting because I can't use the extension.

File: Files/App.xaml (hunk 2 above)
Comment (lukeblevins) [reply Files/App.xaml:111]: @yair100 Those resources weren't available in HighContrast and would crash the app for some reason. It's best to use the correct resources detailed as per the accessibility documentation for Windows apps.

File: Files/UserControls/NavigationToolbar.xaml (hunk 1 above)
Comment (yaichenbaum) [reply Files/UserControls/NavigationToolbar.xaml:324]: Did you make any changes to this file?

Comment (lukeblevins) [reply Files/UserControls/NavigationToolbar.xaml]: @yair100 Yes. I removed an old style and resources from it.

**Review by yaichenbaum — APPROVED** [review]
Nice work! LGTM

File: Files/UserControls/NavigationToolbar.xaml (hunk 1 above)
Comment (yaichenbaum) [reply Files/UserControls/NavigationToolbar.xaml]: Looks good!

**Pull request merged**